lib/internal/util.js

```diff
@@ -187,3 +191,40 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
 
   throw new Error('Unknown signal: ' + signal);
 };
+
+const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
+function promisify(orig) {
+  if (typeof orig !== 'function') {
+    throw new TypeError('The first argument to promisify() must be a function');
```

Comment:
You’re right. I wanted to phrase this error message in a way that makes it as unlikely as possible that changing it is semver-major, but if you prefer another wording, I’ll gladly change it. :smile: